tests/test_functions.py

```diff
@@ -19,6 +20,81 @@ def test_Constant():
     assert isinstance(constant2, Variable)
     assert constant2.__str__() == "5{x}"
 
+    constant1 = Constant(2)
+    constant2 = Constant(7)
+    constant3 = constant1 + constant2
+    assert constant3.__str__() == "{9}"
+
+    constant1 = Constant(2)
+    constant2 = Constant(7)
+    constant3 = constant1 - constant2
+    assert constant3.__str__() == "{-5}"
+
+    constant1 = Constant(5)
+    variable1 = Variable(5, 'x', 3)
+    summation = constant1 + variable1
+    assert summation.__str__() == "{({5}+5{x}^{3})}"
+
+    constant1 = Constant(5)
+    variable1 = Variable(5, 'x', 3)
+    summation = constant1 - variable1
+    assert summation.__str__() == "{({5}-5{x}^{3})}"
+
+    constant1 = Constant(5)
+    constant2 = Constant(5)
+    summation = constant1 * constant2
+    assert summation.__str__() == "{25}"
+
+    constant1 = Constant(5)
+    exp1 = constant1 + Variable(5, 'x', 3)
```

Comment:
Can you define `exp1` explicitly as `Expression` as you did in line 120?
You can make similar changes to lines 55, 63, 72, 88, 94 or add separate cases where you define them explicitly.

Reply:
@8hantanu done